script.rss.editor/changelog.txt

```diff
@@ -1,3 +1,6 @@
+4.0.1
```

Comment:
Please use the `<news>` tag from the addon.xml for this and exclude this file.

Reply:
still here @ronie can you ignore it from the submission?